src/components/Artists/ArtistsItem/ArtistItem.html

```diff
@@ -0,0 +1,13 @@
+<div class="mdc-card artist__card">
+  <div class="artist__card_shaped">
+    <div class="mdc-card mdc-card__primary-action mdc-ripple artist_ripple">
```

Comment:
`artist_ripple` is probably an element -- so it should be `artist__ripple`

Reply:
Fix